msteams-platform/toolkit/debug-local.md

```diff
@@ -0,0 +1,226 @@
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/stop-debugging.png" alt-text="stop debugging":::
+
+
+## Debug locally using Teams Toolkit
+
+After creating a new app using Teams Toolkit:
+
+1. Select **Debug (Edge)** or **Debug (Chrome)** from **Run and Debug** in the activity bar.
+1. Select **Start Debugging (F5)** to run your Teams app in debug mode.
+
+   The following image shows the browser options in the dropdown list:
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/debug.png" alt-text="browser default" border="true"::: 
+
+3. Select **Sign in** to Microsoft 365 account.
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/microsoft365-signin.png" alt-text="Alternate capabilities" border="true":::
+
+   > [!TIP]
+   > You can select **Read more** to learn about Microsoft 365 Developer Program. Your default web browser opens to let you sign in to your Microsoft 365 account using your credentials.
+
+4. Select **Install** for installing the development certificate for localhost.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/debug/install-certificate.png" alt-text="certificate" border="true":::
+
+   > [!TIP]
+   > You can select **Learn More** to know about the development certificate.
+
+5. A dialog box appears depending on your operating system. For Windows, select **Yes**.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/debug/development-certificate.png" alt-text="certification authority" border="true":::
+
+For macOS, in **Certificate Trust Settings** dialog box, enter your **User Name** and **Password**, then select **Update Settings**.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/debug/mac-settings.png" alt-text="mac sign in" border="true":::
+
+After the initial set up process, the Teams Toolkit starts the following processes:
+
+1. [Starts app services](#starts-app-services)
+1. [Launches debuggers](#launches-debuggers)
+1. [Sideloads the Teams app](#sideloads-the-teams-app)
+
+### Starts app services
+
+Runs the tasks defined in `.vscode/tasks.json` as follows:
+
+|  Component |  Task name  | Folder |
+| --- | --- | --- |
+|  Tab |  **Start Frontend** |  tabs |
+|  Bot or messaging extensions |  **Start Bot** |  bot |
+|  Azure Functions |  **Start Backend** |  api |
+
+### Launches debuggers
+
+Launches the debug configurations defined in `.vscode/launch.json` as follows:
+
+:::image type="content" source="../assets/images/teams-toolkit-v2/debug/Terminal.png" alt-text="Start frontend task":::
+
+The table lists debug configuration type for project with tab app and bot app:
+
+|  Component |  Debug configuration name  | Debug configuration type |
+| --- | --- | --- |
+|  Tab |  **Attach to Frontend (Edge)** or Attach to Frontend (Chrome)  |  pwa-msedge or pwa-chrome  |
```

Comment:
Remove bold.

Reply:
I mean remove bold for "Attach to Frontend (Edge)" since "Attach to Frontend (Chrome)" and "Attach to Bot", etc. is not bold.